layouts/footer.slim

```diff
@@ -1,6 +1,35 @@
-footer.ui.center.aligned.container lang='en'
-  hr
-  p Copyright © 2016 by PPType, Inc.
+.ui.footer.black.inverted.segment lang='en'
```

Comment:
Hi, I have changed the `padding-bottom` of the vertical segment to `1em`(default value is 2em), and the `padding-top` of three text column to `1.8em`(previous value is 2em).